src/js/providers/html5.js

```diff
@@ -228,12 +236,15 @@ define([
         }
 
         function _sendMetaEvent() {
+            if (_isAndroidHLS) {
+                return;
+            }
```

Comment:
Not ever sending the meta event could be a problem for integrations that depend on it. How about something like this? On Android I think _duration would still be 0.
```
function _sendMetaEvent() {
    _updateDuration();
    _this.trigger(events.JWPLAYER_MEDIA_META, {
        duration: _duration,
        height: _videotag.videoHeight,
        width: _videotag.videoWidth
    });
}
```